contrib/kind.sh

```diff
@@ -1066,6 +1066,27 @@ docker_create_second_interface() {
   done
 }
 
+docker_create_second_disconnected_interface() {
+  echo "adding second interfaces to nodes"
+  local bridge_name
+  bridge_name="${1:-kindexgw}"
```

Comment:
combine please to one line
you can add `-r`

same for the rest

Reply:
tks. finally decided to not have a subnet on it (since it isn't necessary for now).